console/core/src/main/java/org/eclipse/kapua/app/console/core/client/KapuaCloudConsole.java

```diff
@@ -446,18 +446,23 @@ public void onFailure(Throwable caught) {
 
                     @Override
                     public void onSuccess(final String result) {
-                        logger.info("Waiting for logout.");
-
-                        // this timer is needed to give time to the ConsoleInfo.display method (called above) to show
-                        // the message to the user (otherwise the Window.location.assign would reload the page,
-                        // giving no time to the user to read the message).
-                        Timer timer = new Timer() {
-                            @Override
-                            public void run() {
-                                Window.Location.assign(result);
-                            }
-                        };
-                        timer.schedule(SSO_FAILURE_WAIT_TIME);
+                        if (!result.isEmpty()) {
```

Comment:
Can you please change the header to "2020"? I hope I am not too late 😄

Reply:
Sure! Done